src/renderer/widgets/WalletDetails/ui/ProxiesList.tsx

```diff
@@ -0,0 +1,133 @@
+import { walletProviderModel } from '../model/wallet-provider-model';
+
+type Props = {
+  walletId: ID;
+  canCreateProxy?: boolean;
+  className?: string;
+};
+
+export const ProxiesList = ({ walletId, className, canCreateProxy = true }: Props) => {
+  const { t } = useI18n();
+
+  const proxyAccounts = useUnit(walletProviderModel.$proxyAccounts);
+  const proxyChainGroups = useUnit(proxyModel.$proxyChainGroupStore)[walletId];
+
+  if (!proxyAccounts || !proxyAccounts.length) return <EmptyProxyList className={className} />;
+
+  const proxiesByChain = groupBy(proxyAccounts, 'chainId');
+
+  const forgetProxyAction: DropdownIconButtonOption = {
+    icon: 'forget',
+    title: t('walletDetails.common.removeProxyAction'),
+    onClick: () => {},
+  };
+  const openInfoAction: DropdownIconButtonOption = {
+    icon: 'info',
+    title: t('walletDetails.common.openInfoAction'),
+    onClick: () => {},
+  };
+
+  const getProxyActionButton = (proxyAccount: ProxyAccount, chain: Chain) => {
+    const proxiedAddress = toAddress(proxyAccount.proxiedAccountId, { prefix: chain.addressPrefix });
+
+    return (
+      <DropdownIconButton name="more" className="ml-2">
+        <DropdownIconButton.Items>
+          <DropdownIconButton.Item>
+            <ExplorersPopover
+              address={proxyAccount.accountId}
+              explorers={chain.explorers || []}
+              addressPrefix={chain.addressPrefix}
+              className="-mt-10 -mr-1"
+              button={<DropdownIconButton.Option option={openInfoAction} />}
+            >
+              <ExplorersPopover.Group title={t('walletDetails.common.proxiedAddressTitle')}>
+                <div className="flex items-center gap-x-2">
+                  <HelpText className="text-text-secondary break-all">{proxiedAddress}</HelpText>
+                  <IconButton
+                    className="shrink-0"
+                    name="copy"
+                    size={20}
+                    onClick={() => copyToClipboard(proxiedAddress)}
+                  />
+                </div>
+              </ExplorersPopover.Group>
+            </ExplorersPopover>
+          </DropdownIconButton.Item>
+          {canCreateProxy && (
+            <DropdownIconButton.Item>
+              <DropdownIconButton.Option option={forgetProxyAction} />
+            </DropdownIconButton.Item>
+          )}
+        </DropdownIconButton.Items>
+      </DropdownIconButton>
+    );
+  };
+
+  return (
+    <div className={cnTw('flex flex-col', className)}>
+      <div className="flex items-center px-5 pb-2">
+        <FootnoteText className="flex-1 px-2 text-text-tertiary">{t('accountList.addressColumn')}</FootnoteText>
+      </div>
+
+      <ul className="flex flex-col px-5 divide-y divide-divider overflow-y-auto overflow-x-hidden">
+        {proxyChainGroups.map((chainGroup) => {
+          const { chainId, totalDeposit } = chainGroup;
+
+          const chain = chainsService.getChainById(chainId);
```

Comment:
`getChainById` is not a good function, I'll deprecate it one day
Better to get `networkModel.chains` and `chains[chainId]`